www/app/hardware/Hardware.js

```diff
@@ -2055,7 +2089,37 @@ define(['app'], function (app) {
 					}
 				});
 			}
-			else if (text.indexOf("Buienradar") >= 0) {
```

Comment:
Just to be sure, Buienradar settings are still working ?

Reply:
Everything still works as far as I could see and test :)